redis-tls/reload-redis-certs.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: reload-redis-certs
+  namespace: services
+spec:
+  # every 3 hours at 12 past the hour
+  schedule: "12 */3 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: reload-redis-certs
+            image: alpine:3.19
+            command:
+            - /srv/redis-scripts/reload-certs.sh
+            env:
+            - name: REDISCLI_AUTH
+              valueFrom:
+                secretKeyRef:
+                  name: redis
+                  key: redis-password
+            - name: REDIS_URL
+              valueFrom:
+                configMapKeyRef:
+                  name: redis
+                  key: address
+            volumeMounts:
+            - mountPath: /srv/redis-ca-bundle
+              name: redis-bundle
+              readOnly: true
+            - mountPath: /srv/redis-scripts
+              name: redis-scripts
+              readOnly: true
+          imagePullSecrets:
+          - name: regcred-replicate-production-us
+          restartPolicy: OnFailure
+          volumes:
+          - configMap:
+              defaultMode: 493
```

Comment:
Can we make these octal? Or add a comment with the octal value?

Reply:
making them octal is dangerous, because YAML. See for example https://ansible.readthedocs.io/projects/lint/rules/yaml/#octals

I can add a comment though.